kbcstorage/buckets.py

```diff
@@ -116,7 +116,7 @@ def delete(self, bucket_id, force=False):
         # How does the API handle it when force == False and the bucket is non-
         # empty?
         url = '{}/{}'.format(self.base_url, bucket_id)
-        params = {'force': force}
+        params = {'force': force, 'async': asynchronous}
```

Comment:
Afaik this doesn't make any sense - either it needs to handle the async wait (preferably), or return job id so that it can be done manually outside